pkg/utils/events.go

```diff
@@ -27,5 +29,7 @@ const (
 	EventMessageFenceAgentExecuted       = "Fence agent was executed"
 	EventMessageFenceAgentSucceeded      = "Fence agent was succeeded"
 	EventMessageDeleteResources          = "Manually delete pods from the unhealthy node"
+	EventMessageAddOutOfServiceTaint     = "out-of-service taint was added"
```

Comment:
nit: please align with other messages and start with an uppercase letter (same below)

Reply:
Fixed it.